### DISCUSSION TITLE
Allow contained items to have collisions when " hide="false" "

### DISCUSSION THREAD

**Discussion by NotAlwaysTrue** [FakeFishGames/Barotrauma#13653] [Ideas and suggestions]
This feature will allow many interesting thinkings, for example, a gun-mounted shield. You can also make a shield sword :).

**Comment by jigardi** [comment]
Have my upvote!
This sounds extremely useful.  If I'm not mistaken, this could be used to create weakspots on creatures/machinery and possibly even wearables.
